src/scss/custom/modules/_checkout.scss

```diff
@@ -0,0 +1,10 @@
+.payment__option {
```

Comment:
This shouldn't be inside core of Hummingbird but rather inside your theme or ps_checkout module :)